docs/api/covidcast-signals/safegraph.md

```diff
@@ -4,20 +4,26 @@ parent: Data Sources and Signals
 grand_parent: COVIDcast API
 ---
 
-# SafeGraph Mobility
+This data source uses data reported by [SafeGraph](https://www.safegraph.com/)
+using anonymized location data from mobile phones. Safegraph provides several different datasets to eligible researchers. We surface signals from two such datasets.
+# Table of contents
```

Comment:
A formatting comment: There should be a title at the top of the page labeling this as the SafeGraph source, in line with the `title` field:

```
# SafeGraph
{: .no_toc}
```

The table of contents and all headings below should be one level deeper -- that is, the table of contents and the two big SafeGraph headings should be `##`, and anything under them should be `###`.

That matches how we do the other source pages and makes the Markdown formatting look right

Reply:
Thanks. I have updated it. Can you help check whether it seems correct now?